browser/js/config/configureStore.js

```diff
@@ -8,12 +8,13 @@ import rootReducer from '../reducers';
 const loggerMiddleware = createLogger();
 
 export default function configureStore (initialState) {
+  const middlewares = [thunkMiddleware]
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
Is this working? This file seems like it's going to get executed in the browser context and would not have access to `process`.